requirements.txt

```diff
@@ -2,26 +2,18 @@
 -e .
 
 # external requirements
-black==19.3b0
-dvc[s3]==0.84.0
+black==19.10b0
```

Comment:
I'd be happier to remove frozen versions from these deps as much as possible I guess.

Reply:
Happy to talk about it, but this is probably a separate discussion from this PR.